### PR TITLE
Other media meta settings processing

### PR DESCRIPTION
**Pull request by bschuiling**
See my remarks in Asana.

### CODE REVIEW

File: class/db/shortpixel-custom-meta-dao.php [hunk 1]
```diff
@@ -292,6 +292,7 @@ protected function metaToParams($meta) {
     public function addImage($meta) {
         $p = $this->metaToParams($meta);
         $id = $this->db->insert($this->db->getPrefix().'shortpixel_meta', $p->params, $p->types);
+        exit('insert action done. DAO');
```
Comment (short-pixel-optimizer) [inline class/db/shortpixel-custom-meta-dao.php]: Should this exit remain here?

Comment (bschuiling) [reply class/db/shortpixel-custom-meta-dao.php]: Ah woops no, good catch. Removed this.

File: class/model/shortpixel-meta.php [hunk 2]
```diff
@@ -32,30 +32,35 @@ class ShortPixelMeta extends ShortPixelEntity{
     protected $tsAdded;
     protected $tsOptimized;
     protected $thumbs;
-    
+
```
Comment (short-pixel-optimizer) [inline class/model/shortpixel-meta.php]: :+1: Most needed...

File: class/model/shortpixel-meta.php [hunk 3]
```diff
@@ -32,30 +32,35 @@ class ShortPixelMeta extends ShortPixelEntity{
...
     const TABLE_SUFFIX = 'meta';
     const WEBP_THUMB_PREFIX = 'sp-webp-';
     const FOUND_THUMB_PREFIX = 'sp-found-';
-    
+
+    // [BS] Attempt to shed some light on Meta Status on File.
+    const FILE_STATUS_UNPROCESSED = 0;
+    const FILE_STATUS_PENDING = 1;
+    const FILE_STATUS_SUCCESS = 2;
+
```
Comment (short-pixel-optimizer) [inline class/model/shortpixel-meta.php]: :+1: Most needed...

**Pull request merged**